integrationtests/tests/tests.rs

```diff
@@ -29,7 +29,9 @@ use mint_client::ClientError;
 async fn peg_in_and_peg_out_with_fees() {
     let peg_in_amount: u64 = 5000;
     let peg_out_amount: u64 = 1200; // amount requires minted change
-    let (fed, user, bitcoin, _, _) = fixtures(2, &[sats(10), sats(100), sats(1000)]).await;
+    let (fed, user, bitcoin) = Fixtures::new(2, &[sats(10), sats(100), sats(1000)])
```

Comment:
A bit clean to remove the `async` see https://github.com/NicolaLS/minimint/pull/11